Sources/Danger/Plugins/SwiftLint/SwiftLint.swift

```diff
@@ -4,42 +4,71 @@ import Foundation
 /// The SwiftLint plugin has been embedded inside Danger, making
 /// it usable out of the box.
 public enum SwiftLint {
+    public enum LintStyle {
+        case all(directory: String?)
+        case modifiedAndCreatedFiles(directory: String?)
+        case files([File])
+    }
+
     static let danger = Danger()
     static let shellExecutor = ShellExecutor()
 
+    /// This method is deprecated in favor of
+    @available(*, deprecated, message: "Use the lint(_ lintStyle ..) method instead.")
+    @discardableResult
+    public static func lint(inline: Bool = false,
+                            directory: String? = nil,
+                            configFile: String? = nil,
+                            strict: Bool = false,
+                            quiet: Bool = true,
+                            lintAllFiles: Bool = false,
+                            swiftlintPath: String? = nil) -> [SwiftLintViolation] {
+        let lintStyle: LintStyle = lintAllFiles ? .all(directory: directory) : .modifiedAndCreatedFiles(directory: directory)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/Plugins/SwiftLint/SwiftLint.swift;
  Line: 26;
-->

- :warning: Line should be 120 characters or less: currently 125 characters (`line_length`)